src/modules/fw_att_control/FixedwingAttitudeControl.cpp

```diff
@@ -726,40 +734,44 @@ void FixedwingAttitudeControl::control_flaps(const float dt)
 	}
 
 	// move the actual control value continuous with time, full flap travel in 1sec
-	if (fabsf(_flaps_applied - flap_control) > 0.01f) {
-		_flaps_applied += (_flaps_applied - flap_control) < 0 ? dt : -dt;
+	_flaps_setpoint_with_slewrate.update(flap_control, dt);
+}
 
-	} else {
-		_flaps_applied = flap_control;
-	}
+void FixedwingAttitudeControl::control_spoilers(const float dt)
+{
+	float spoiler_control = 0.f;
 
-	/* default flaperon to center */
-	float flaperon_control = 0.0f;
+	if (_vcontrol_mode.flag_control_manual_enabled) {
+		switch (_param_fw_spoilers_man.get()) {
+		case 0:
+			break;
 
-	/* map flaperons by default to manual if valid */
-	if (PX4_ISFINITE(_manual_control_setpoint.aux2) && _vcontrol_mode.flag_control_manual_enabled
-	    && fabsf(_param_fw_flaperon_scl.get()) > 0.01f) {
+		case 1:
```

Comment:
after a quick look, im guessing the answer to this is no, so this is probably a discussion for another issue/PR - but is there currently functionality in the param system to get any enumeration? (e.g. set from the `@value`) - would be nice to pull the enums directly from the param definition and use them in the code. e.g. something like `case param_fw_spoilers_man_::kFlapsChannel:`

Reply:
I don't think it's possible atm, but would be cool to have. @bkueng am I right that it doesn't exist atm?